src/main/java/com/fvv/bookstore/menu/Menu.java

```diff
@@ -422,6 +429,93 @@ private void createSubMenuEmployee() {
 		}
 	}
 	
+	/**
+	 * Method to create and appear the reports sub menu.
+	 */
+	private void createSubMenuReports() {
+		while(true) {
+			String input = JOptionPane.showInputDialog(
+					null, "Reports Section" + Constants.LINE_SEPARATOR + Constants.LINE_SEPARATOR +
+					"Select an option below:" + Constants.LINE_SEPARATOR +
+					"1 - Individuals" + Constants.LINE_SEPARATOR +
+					"2 - Products - Stock to Replace" + Constants.LINE_SEPARATOR +
+					"3 - Sales" + Constants.LINE_SEPARATOR +
+					"0 - Return",
+					"RAINBOW BOOKSTORE", JOptionPane.PLAIN_MESSAGE
+			);
+			switch(input.charAt(0)) {
+				case '1' : this.createSubMenuIndividualsReports();
+					break;
+				case '2' : this.listStockToReplace();
+					break;
+				case '3' : this.createSubMenuSalesReports();
+					break;
+				case '0' : this.createMenu();
+					break;
+				default : JOptionPane.showMessageDialog(null, "Choose a valid option!", 
+						"Error", JOptionPane.ERROR_MESSAGE);
+			}
+		}
+	}
+	
+	/**
+	 * Method to create and appear the individuals reports sub menu.
+	 */
+	private void createSubMenuIndividualsReports() {
+		while(true) {
+			String input = JOptionPane.showInputDialog(
+					null, "Individuals Reports Section" + Constants.LINE_SEPARATOR + Constants.LINE_SEPARATOR +
+					"Select an option below:" + Constants.LINE_SEPARATOR +
+					"1 - Customer Filtered and Ordered by Name" + Constants.LINE_SEPARATOR +
```

Comment:
it must be "Customers", plural, you are getting more than one customer here (apply for all messages related to this report)

Reply:
ok